schema/pgsql/schema.sql

```diff
@@ -90,45 +90,71 @@ CREATE TABLE schedule (
     CONSTRAINT pk_schedule PRIMARY KEY (id)
 );
 
+CREATE TABLE rotation (
+    id bigserial,
+    schedule_id bigint NOT NULL REFERENCES schedule(id),
+    -- the lower the more important, starting at 0, avoids the need to re-index upon addition
+    priority integer NOT NULL,
+    name text NOT NULL,
+    mode rotation_type NOT NULL,
+    -- JSON with rotation-specific attributes
+    -- Needed exclusively by Web to simplify editing and visualisation
+    options text NOT NULL,
+
+    -- A date in the format 'YYYY-MM-DD' when the first handoff should happen.
+    -- It is a string as handoffs are restricted to happen only once per day
+    first_handoff date NOT NULL,
+
+    -- Set in case the first_handoff was in the past during creation of the rotation.
+    -- It is essentially the creation time of the rotation.
+    -- Used by Web to avoid showing shifts that never happened
+    actual_handoff bigint,
```

Comment:
> It is essentially the creation time of the rotation.

Even if `first_handoff` is set to be in the future when creating the rotation?

Reply:
No, in that case it's NULL

Reply:
Can you always set it? That would actually make it useful as it would allow to easily determine the currently active version of the rotation (the one with the largest value of `actual_handoff`, that is at or before the current time).

Reply:
Only if the switchover to the newer version happens at a fixed time like 00:00 UTC. Otherwise, it misses timezone and/or time of day information.

Reply:
Changed it, by the way